x-pack/test/send_search_to_background_integration/tests/apps/dashboard/async_search/sessions_in_space.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+import { getSearchSessionIdByPanelProvider } from './get_search_session_id_by_panel';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const testSubjects = getService('testSubjects');
+  const esArchiver = getService('esArchiver');
+  const security = getService('security');
+  const PageObjects = getPageObjects([
+    'common',
+    'header',
+    'dashboard',
+    'visChart',
+    'security',
+    'timePicker',
+  ]);
+  const getSearchSessionIdByPanel = getSearchSessionIdByPanelProvider(getService);
+  const browser = getService('browser');
+  const sendToBackground = getService('sendToBackground');
+
+  describe('dashboard in space', () => {
```

Comment:
I'd love to see a similar test added for discover 🙏

Reply:
WIll do in a separate PR :)

Reply:
I think it's worth re-iterating the following section from our development best practices:

> Hide your feature behind a feature flag

> An example here is Chat Ops. The Chat Ops MVP is broken down into five implementation phases. We wouldn't release the feature without all of those five phases, but the developers could check each phase into master, behind a feature flag.

> Because this is code that is being merged directly into master, all PRs should still be accompanied by tests and whatever functionality is exposed, should be complete and bug free.

> A good question to ask yourself is, how will you feel if a customer turns this feature on? Is it usable, even if not up to the level of something we would market? It should have some level of minimal utility.

> Another question to ask yourself is, if this feature gets cancelled, how difficult will it be to remove?

If we're merging code to master, even if it's behind a feature flag, we should **not** be skipping tests.

Reply:
Agreed!